packages/elements-react/src/components/form/form.tsx

```diff
@@ -37,46 +37,147 @@ import {
+     *
+     * It is used to show the current identifier and can allow the user to start a new flow, if they want to.
+     */
+    CurrentIdentifierButton: ComponentType<OryCurrentIdentifierProps>
+    /**
+     * Anchor component, rendered whenever an "anchor" node is encountered
+     */
+    Anchor: ComponentType<OryNodeAnchorProps>
+    /**
+     * The Input component is rendered whenever a "input" node is encountered.
+     */
+    Input: ComponentType<OryNodeInputProps>
+    /**
+     * Special version of the Input component for OTP codes.
+     */
+    CodeInput: ComponentType<OryNodeInputProps>
+    /**
+     * The Image component is rendered whenever an "image" node is encountered.
+     *
+     * For example used in the "Logo" node.
+     */
+    Image: ComponentType<OryNodeImageProps>
+    /**
+     * The Label component is rendered around Input components and is used to render form labels.
+     */
+    Label: ComponentType<OryNodeLabelProps>
+    /**
+     * The Checkbox component is rendered whenever an input node with of boolean type is encountered.
+     */
+    Checkbox: ComponentType<OryNodeInputProps>
+    /**
+     * The Text component is rendered whenever a "text" node is encountered.
+     */
+    Text: ComponentType<OryNodeTextProps>
+  }
+  Card: {
+    /**
+     * The card container is the main container of the card.
+     */
+    Root: ComponentType<OryCardRootProps>
+    /**
+     * The card footer is the footer of the card container.
+     */
+    Footer: ComponentType<OryCardFooterProps>
+    /**
+     * The card header is the header of the card container.
+     */
+    Header: ComponentType<OryCardRootProps>
+    /**
+     * The card content is the main content of the card container.
+     */
+    Content: ComponentType<OryCardContentProps>
+    /**
+     * The card logo is the logo of the card container.
+     */
+    Logo: ComponentType
+    /**
+     * The HorizontalDivider component is rendered between groups.
+     */
+    Divider: ComponentType<OryCardDividerProps>
+    /**
+     * The AuthMethodListItem component is rendered on the "method" chooser step in the identifier_first login flow.
+     *
+     * This is only used, if login is configured to use identifier_first authentication.
+     */
+    AuthMethodListItem: ComponentType<OryCardAuthMethodListItemProps>
+  }
+  Form: {
+    /**
+     * The FormContainer component is the main container of the form.
+     *
+     * It should render its children.
+     *
+     * You most likely don't want to override this component directly.
+     */
+    Root: ComponentType<OryFormRootProps>
+    /**
+     * A special form group container for the social buttons.
+     *
+     * This is required, because the social buttons need to be in its form, to not influence the other form groups.
+     *
+     * You most likely don't want to override this component directly.
+     */
+    OIDCRoot: ComponentType<OryFormOIDCRootProps>
```

Comment:
I think we could rather do `OidcRoot`

Reply:
>When using acronyms, use Pascal case or camel case for acronyms more than two characters long. For example, use HtmlButton or htmlButton. However, you should capitalize acronyms that consist of only two characters, such as System.IO instead of System.Io.

So yes, good point. Will do.

Reply:
Done.